engine/source/output/anim/generate/output_div_u.F

```diff
@@ -0,0 +1,367 @@
+Copyright>        but WITHOUT ANY WARRANTY; without even the implied warranty of
+Copyright>        MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+Copyright>        GNU Affero General Public License for more details.
+Copyright>
+Copyright>        You should have received a copy of the GNU Affero General Public License
+Copyright>        along with this program.  If not, see <https://www.gnu.org/licenses/>.
+Copyright>
+Copyright>
+Copyright>        Commercial Alternative: Altair Radioss Software
+Copyright>
+Copyright>        As an alternative to this open-source version, Altair also offers Altair Radioss
+Copyright>        software under a commercial license.  Contact Altair to discuss further if the
+Copyright>        commercial version may interest you: https://www.altair.com/radioss/.
+Chd|====================================================================
+Chd|  SCHLIEREN                     source/output/anim/generate/schlieren.F
+Chd|-- called by -----------
+Chd|        DFUNC0                        source/output/anim/generate/dfunc0.F
+Chd|        DFUNCC                        source/output/anim/generate/dfuncc.F
+Chd|        DFUNCS                        source/output/anim/generate/dfunc6.F
+Chd|        H3D_QUAD_SCALAR               source/output/h3d/h3d_results/h3d_quad_scalar.F
+Chd|        H3D_SHELL_SCALAR_1            source/output/h3d/h3d_results/h3d_shell_scalar_1.F
+Chd|        H3D_SOLID_SCALAR_1            source/output/h3d/h3d_results/h3d_solid_scalar_1.F
+Chd|-- calls ---------------
+Chd|        ALE_CONNECTIVITY_MOD          ../common_source/modules/ale/ale_connectivity_mod.F
+Chd|        ELBUFDEF_MOD                  ../common_source/modules/mat_elem/elbufdef_mod.F
+Chd|        INITBUF_MOD                   share/resol/initbuf.F         
+Chd|====================================================================
+      SUBROUTINE OUTPUT_DIV_U(
+     1                  EVAR   , IX         ,X      ,V      ,
+     2                  IPARG  , ELBUF_TAB  ,VOL    ,
+     3                  NG     , NIX        ,ITYP    )
+C-----------------------------------------------
+C   D e s c r i p t i o n
+C-----------------------------------------------
+C     This subroutine outputs data for div(v) : volumetric dilatation rate.
+C-----------------------------------------------
+C   P r e - C o n d i t i o n s
+C-----------------------------------------------
+C     IALEL > 0 
+C        where IALEL =IPARG(7,NG)+IPARG(11,NG)
+C-----------------------------------------------
+C   M o d u l e s
+C-----------------------------------------------
+      USE INITBUF_MOD
+      USE ELBUFDEF_MOD
+C-----------------------------------------------
+C   I m p l i c i t   T y p e s
+C-----------------------------------------------
+#include      "implicit_f.inc"
+C-----------------------------------------------
+C   C o m m o n   B l o c k s
+C-----------------------------------------------
+#include      "com01_c.inc"
```

Comment:
@Schwoertzig 

use of include files is no more allowed in new routine .

Reply:
Hi @sebastienVilleneuve , includes directives are now removed.

Reply:
Thanks a lot Thierry